plugins/main/public/kibana-integrations/kibana-discover.js

```diff
@@ -1127,7 +1127,7 @@ function discoverController(
   $scope.$watch('fetchStatus', () => {
     if ($scope.fetchStatus !== fetchStatuses.UNINITIALIZED) {
       setTimeout(() => {
-        modulesHelper.hideCloseButtons();
+        //modulesHelper.hideCloseButtons();
```

Comment:
if nothing is going to be done in the timeout, it may be possible to delete the conditional

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6534/commits/904c03fd54eaa3db21311224b92d1273ee371d23